src/Symfony/Component/Ldap/Adapter/ExtLdap/Query.php

```diff
@@ -177,10 +168,10 @@ public function getResources(): array
     /**
      * Resets pagination on the current connection.
      */
-    private function resetPagination()
+    protected function resetPagination()
```

Comment:
@nicolas-grekas do you think I should mark it private again ? Same for `controlPagedResult`, `controlPagedResultResponse` and `callSearchFunction` ?

Reply:
all methods should be private yes